modules/nw-proxy-configure-object.adoc

```diff
@@ -117,3 +117,5 @@ authority from the RHCOS trust bundle.
 --
 
 . Save the file to apply the changes.
+
+NOTE: The URL scheme must be http; https is currently not supported.
```

Comment:
I would avoid the semicolon here for clarity, especially since it is often followed by a colon. And for readability, it would be good to wrap `http` and `https` in backticks as we do in the previous callout notes. 

Perhaps one of these options instead?

- The URL scheme must be `http` as `https` is currently not supported.
- The URL scheme must be `http`. The `https` scheme is currently not supported.

Reply:
Ah, ok. I was looking at a note earlier in this file and copied that. Should I add the brackets and ==== to the others as well? Or maybe there's a difference between the two I'm not grasping? And added that other correction.

Reply:
You should wrap `http` and `https` in backticks here for readability as values. And I think it should say "the `https` scheme..." (add "scheme"); otherwise, it seems to be missing a noun.

And if you wouldn't mind updating with the `====` formatting elsewhere in this file, too, that would be great! I didn't catch that. Thanks!!